domain_source/domain_source.module

```diff
@@ -166,7 +169,7 @@ function domain_source_form_validate($element, \Drupal\Core\Form\FormStateInterf
 /**
  * Redirect form submissions to other domains.
  */
-function domain_source_form_submit(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+function domain_source_form_submit(&$form, FormStateInterface $form_state) {
```

Comment:
Do we have to type hint &$form?

Reply:
FormSubmitterInterface::doSubmitForm does not typehint the &$form array, but it is assumed to be an array so I added a type cast to make this explicit